# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/esden/povray31/distribution_message.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./source/optout.h~	Mon Nov 22 04:24:23 1999
+++ ./source/optout.h	Thu Sep 12 05:57:41 2002
@@ -36,7 +36,7 @@
 #define POV_RAY_VERSION "3.1g"
 
 #define DISTRIBUTION_MESSAGE_1 "This is an unofficial version compiled by:"
-#define DISTRIBUTION_MESSAGE_2 "FILL IN NAME HERE........................."
+#define DISTRIBUTION_MESSAGE_2 "--- the ROCK Linux build scripts ... ( http://www.rocklinux.org ) ---"
 #define DISTRIBUTION_MESSAGE_3 "The POV-Ray Team(tm) is not responsible for supporting this version."
 
 /* Number of help pages (numbered 0 to MAX_HELP_PAGE). */
